examples/logging.yml

```diff
@@ -24,6 +24,17 @@ services:
   - name: write-to-the-logs
     image: alpine
     command: ["/bin/sh", "-c", "while /bin/true; do echo hello $(date); sleep 1; done" ]
+  - name: write-and-rotate-logs
+    image: linuxkit/logwrite:80b8619e0a2e9ffaa4ec314886b2606a839a094e
+    binds:
+      - /var/run:/var/run
+      - /var/log:/var/log
+  - name: kmsg
+    image: linuxkit/kmsg:862384637bd541d499b2f1356f71a1d304f79cb5
+    binds:
```

Comment:
can you add the binds and caps to the `build.yml` for `kmsg`. They are unlikely to change and keep the YAMLs cleaner. Look at `pkg/getty/build.yml` for examples.